sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_properties.asciidoc

```diff
@@ -486,6 +496,10 @@ units that declare that same function _F_ must list the same set of properties
 _P_ via the `SYCL_EXT_ONEAPI_FUNCTION_PROPERTY` macro. Programs which fail to do this
 are ill-formed, but no diagnostic is required.
 
+NOTE: Due to a restriction on attribute ordering in Clang it is only currently
+possible to use `SYCL_EXT_ONEAPI_FUNCTION_PROPERTY` before `SYCL_EXTERNAL` in
+{dpcpp}.
```

Comment:
There is no definition of the variable `{dpcpp}`, so this is not rendered correctly in HTML.  Either add a definition or use `DPC++` here.